### DISCUSSION TITLE
Is there a way avoid repeated `@Mapping(target = "foo", ignore = true)` annotations?

### DISCUSSION THREAD

**Discussion by efenderbosch-atg** [mapstruct/mapstruct#3537] [Q&A] [answered]
Instead of a bunch of repeated annotations like this:
```kotlin
    @Mapping(target = "foo", ignore = true)
    @Mapping(target = "bar", ignore = true)
    @Mapping(target = "baz", ignore = true)
    @Mapping(target = "qux", ignore = true)
```
Something similar to `@BeanMapping::ignoreUnmappedSourceProperties`?

```kotlin
    @BeanMapping(ignoreTargets = ["foo", "bar", "baz", "qux"])
````

**Comment by filiphr** [comment, marked as answer]
I am not sure how I feel about this. `Mapping#target` supports nesting. We could in theory "remap" `BeanMapping#ignoreTarget` to multiple `Mapping@ignore` and use them, but not sure how easy / difficult it would be to achieve that on our side. Perhaps something worth exploring. I've created #3539 to explore this.
